js/questionset.js

```diff
@@ -706,7 +707,7 @@ H5P.QuestionSet = function (options, contentId, contentData) {
     var finals = self.getScore();
     var totals = self.getMaxScore();
 
-    var scoreString = H5P.Question.determineOverallFeedback(params.endGame.overallFeedback, finals / totals).replace('@score', finals).replace('@total', totals);
+    var scoreString = H5P.Question.determineOverallFeedback(params.endGame.overallFeedback, finals / totals).replace('@score', finals).replace('@total', totals) ?? undefined;
```

Comment:
@larsmagu Beware of nullish coalescing without using Babel. Nullish coalescing is not supported by Safari 13- (current version is 15), and the H5P core team strives to support the latest three browser versions if that policy has not changed.